### PR TITLE
Add weather app example to api.md

### PR DESCRIPTION
**Pull request by ntflix**

### CODE REVIEW

File: docs/tildagon-apps/api.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ title: Use an API in an app
...
 ---
 
+
 !!! tip "You've found a badge CHALLENGE!"
 
     Your challenge, should you choose to accept it, is to create a minimal example app that shows how you can call an API from an app. To see more information and accept the challenge (that is, comment on the issue), see this [issue](https://github.com/emfcamp/badge-2024-documentation/issues/69).
+
+## Example apps
+
+- [A simple example to fetch and display current weather at EMFCamp location](https://github.com/ntflix/Tildagon-Weather).
```
Comment (hughrawlinson) [inline docs/tildagon-apps/api.md:13]: This isn't really a self-contained example of how to do this specific thing - in other words, this app does more than just demonstrate how to call an API.

Comment (ntflix) [reply docs/tildagon-apps/api.md:13]: Could you let me know what more it does than is necessary for a demo of calling an API please, @hughrawlinson ?

Comment (npentrel) [reply docs/tildagon-apps/api.md:13]: Absolutely no need to trim down this is excellent! I'll test it now and add it - for the docs I may merge it into one file but I will do that :)

**Comment by npentrel** [comment]
@ntflix thank you again - this is a great example. I've made some changes to the app to display it in one file in the docs https://github.com/emfcamp/badge-2024-documentation/pull/91 . I've also adjusted how some of it works because I saw a blank screen for a while when booting up. I won't add this to your app because it changes it more than a little but if you want the changes please feel free to add them!

**Pull request merged**